src/handlers.rs

```diff
@@ -124,6 +124,29 @@ pub async fn handle(ctx: &Context, event: &Event) -> Vec<HandlerError> {
     errors
 }
 
+async fn validate_triagebot(ctx: &Context, event: &IssuesEvent, errors: &mut Vec<HandlerError>) {
```

Comment:
As far as I can see this function would inject an HTTP call in the `issue_handlers` macro which is called quite often. I am a bit worried about the additional overhead on every issue the triagebot is called to handle.

What is your goal here @meysam81 ? validate the triagebot config for a certain repository, right? Can this check happen _just once_ on triagebot startup?

Also the name of this function could be a bit more explicit about its purpose (e.g. `validate_triagebot_config` or something similar).

Reply:
This function should be placed in its own module at minimum (similar to all the other handlers). I'd emulate something like rustc_commits in terms of structure.

I would expect that if the goal is to validate PRs, then we would do this only on push events. It probably also makes sense to delegate the actual validation to the config module, not just calling `toml::from_slice` here.